src/main/webapp/WEB-INF/view/login.jsp

```diff
@@ -18,6 +18,12 @@
 <head>
   <title>Login</title>
   <link rel="stylesheet" href="/css/main.css">
+  <style>
+    label {
```

Comment:
Any style changes should be in a css file. Ideally we'd have a different css file for each url but I'll leave that up to you.